python/dgl/graphbolt/minibatch_sampler.py

```diff
@@ -0,0 +1,104 @@
+"""Minibatch Sampler"""
+
+from typing import Mapping, Optional
+
+from torch.utils.data import default_collate
+from torchdata.datapipes.iter import IterableWrapper, IterDataPipe
+from .itemset import *
+from ..batch import batch as dgl_batch
+from ..heterograph import DGLGraph
+
+__all__ = ["MinibatchSampler"]
+
+
+def _colloate(batch):
```

Comment:
typo:
coalesce? collate?
```suggestion
def _collate(batch):
```

Reply:
no, it's `collate` just as the datapipe function name: `default_collate`

Reply:
I see, but still typo.